files/asus_generic/features.txt

```diff
@@ -5,3 +5,4 @@ rescan_devices
 hardware_content_filtering
 mac_filter
 edns0_content_filter
+gzip
```

Comment:
Let's make the feature name `gzip_requests` - to match the command line and config option.